firefox-ios/firefox-ios-tests/Tests/XCUITests/InactiveTabsTest.swift

```diff
@@ -0,0 +1,137 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class InactiveTabsTest: BaseTestCase {
+    override func setUp() {
+        launchArguments = [
+            LaunchArguments.PerformanceTest,
+            LaunchArguments.SkipIntro,
+            LaunchArguments.SkipWhatsNew,
+            LaunchArguments.SkipETPCoverSheet,
+            LaunchArguments.SkipContextualHints,
+            LaunchArguments.DisableAnimations
+        ]
+
+        // Load 20 tabs. 19 tabs are inactive.
+        let tabsDatabase = "tabsState20.archive"
+        launchArguments.append(LaunchArguments.LoadTabsStateArchive + tabsDatabase)
+        launchArguments.append(LaunchArguments.LoadDatabasePrefix + tabsDatabase)
+
+        super.setUp()
+
+        // Workaround: Restart app to guarantee the tabs are loaded
+        waitForTabsButton()
+        waitUntilPageLoad()
+        app.terminate()
+        sleep(2)
```

Comment:
Can we remove this? the app will terminate and then start again when it is ready

Reply:
Removed. We don't need the `sleep()`